test/bin/scenario.sh

```diff
@@ -1188,6 +1202,13 @@ action_run() {
 
     check_dependencies
 
+    for hname in "$@"; do
+	    if [[ ! "${hname}" == "host1" ]]; then
```

Comment:
We cannot assume `host1` by default. There are tests that are using different host names.

I would simply check the number of the command line arguments. If there's 3 - assume the third one is the override host name or IP.

Reply:
when i try to print $@ it just shows only 1 argument which is either host1 or the ip, am i missing something here ?

Reply:
It depends on how you run the scenario.sh script. We're always removing action and script arguments [here](https://github.com/openshift/microshift/blob/main/test/bin/scenario.sh#L1227-L1230).

Next, we're passing the remaining command line to `action_xyz` handers. 

So, if you run `scenario.sh run myscenario.sh my_ip` command, the `my_ip` will be passed to `action_run`.

Reply:
Most of the scenario scripts use host1, but look at [this one](https://github.com/openshift/microshift/blob/main/test/scenarios-bootc/periodics/el96-src%40cncf-conformance.sh#L172-L173) for example - it creates more than one host.

Reply:
this is how i am trying to run `scenario.sh run myscenario.sh my_ip` and to handle the other case that you are talking about here would using a regex instead of `host1` work ?

Reply:
It's hard for me to comment without seeing the new code. Maybe, you can upload changes and I will take another look?

Reply:
sure, will add in sometime, thanks !!